app/templates/index-g-cloud.html

```diff
@@ -27,19 +31,47 @@ <h1>
       {{ title|capitalize }}
     </h1>
   </header>
-  <div class="grid-row">
-    <form action="{{ url_for('main.search_services') }}" method="get" class="search-box column-two-thirds">
-      <input type="text" class="text-box" name="q" maxlength="200">
-      <button type="submit" class="button-save">
-        Show services
-      </button>
-    </form>
-  </div>
   <div class="grid-row">
     <div class="column-two-thirds framework-lots">
```

Comment:
Don't think we need the `framework-lots` class anymore, as there's no longer a `ul` inside this div. I think this means we can get rid of it in `_gcloud_browse_list.scss` as well 😸